docs/v3/how-to-guides/automations/chaining-deployments-with-events.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Chaining deployments with events
+description: Learn how to establish event-based dependencies between individual workflows.
+---
+
+### Trigger a downstream deployment with an event
+
+This example shows how to use a trigger to schedule a downstream deployment when an upstream deployment completes.
```

Comment:
I think we need to find consistent language when talking about runs created from a deployment.
```suggestion
This example shows how to use a trigger to schedule a run of downstream deployment when the run of an upstream deployment completes.
```